packages/eslint-plugin-template/tests/configs.test.ts

```diff
@@ -26,7 +29,21 @@ describe('configs', () => {
         Object.keys(eslintPluginTemplate.rules).every((ruleName) =>
           containsRule(eslintPluginTemplate.configs.all, ruleName),
         ),
-      );
+      ).toBe(true);
+    });
+
+    it('should only contain valid rules', () => {
+      expect(
+        Object.keys(eslintPluginTemplate.configs.all.rules)
+          .filter((ruleName) => ruleName.startsWith(ESLINT_PLUGIN_PREFFIX))
+          .every((ruleName) =>
+            Boolean(
+              (eslintPluginTemplate.rules as any)[
```

Comment:
You could use `keyof typeof eslintPlugin.rules` to avoid `any` (here and in other places):

```suggestion
              (eslintPlugin.rules as keyof typeof eslintPlugin.rules)[
```